packages/api/src/services/location.ts

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+import { GeoLocation } from '@sf/core/location';
+
+export interface LocationService {
+  getGeoLocation(address: string): Promise<GeoLocation>;
+}
+
+export const locationService: LocationService = new (class implements LocationService {
+  async getGeoLocation(address: string): Promise<GeoLocation> {
```

Comment:
How would you implement it if the stakeholders want to have multiple map providers?

Reply:
Surely make different services to handle `mapbox` or `NASA` or `google map` apis.
Each providers will have the same interface but different implementations.